src/H5Cprivate.h

```diff
@@ -880,6 +977,8 @@ typedef herr_t (*H5C_serialize_func_t)(const H5F_t *f, void *image_ptr, size_t l
 typedef herr_t (*H5C_notify_func_t)(H5C_notify_action_t action, void *thing);
 typedef herr_t (*H5C_free_icr_func_t)(void *thing);
 typedef herr_t (*H5C_get_fsf_size_t)(const void *thing, hsize_t *fsf_size_ptr);
+typedef herr_t (*H5C_vfd_swmr_refresh_func_t)(H5F_t *f, void *entry_ptr, const void *image_ptr,
```

Comment:
This should not mention the 'vfd_swmr' and should just be named 'H5C_refresh_func_t' - it may be used for other purposes in the future.

Reply:
Fixed